Automattic/newspack-plugin
# feat(ia): update audience links in newspack dashboard

## NB: this PR depends on https://github.com/Automattic/newspack-plugin/pull/3565 merging first.

### All Submissions:


### Changes proposed in this Pull Request:

Add Audience related links to Newspack > Dashboard.

### How to test the changes in this Pull Request:

1. Checkout this branch.
2. Navigate to _/wp-admin/admin.php?page=newspack-dashboard_
3. Scroll to "Audience development".
![Screenshot 2024-12-12 at 11 26 51](https://github.com/user-attachments/assets/8668cd7d-37fd-4902-b127-4662b8849541)
4. Click each card and confirm each card navigates to the correct Audience wizard/page.

### Other information:
